test/basic.js

```diff
@@ -53,7 +53,7 @@ t.test('spawn execPath', function (t) {
 })
 
 t.test('exec shebang', function (t) {
-  var child = cp.exec(fixture + ' xyz')
+  var child = spawn(fixture, ['xyz'])
```

Comment:
This should still be cp.exec.

The failures on Linux can probably be fixed with this diff:

```diff
diff --git a/test/fixtures/script.js b/test/fixtures/script.js
index 51f518a..ebc08f9 100755
--- a/test/fixtures/script.js
+++ b/test/fixtures/script.js
@@ -1,3 +1,5 @@
 #!/usr/bin/env node
 console.log('%j', process.execArgv)
 console.log('%j', process.argv.slice(2))
+// keep the process open long enough to catch a signal
+setTimeout(function() {}, 500)
```